template-tags.php

```diff
@@ -553,7 +553,7 @@ function coauthors_wp_list_authors( $args = array() ) {
 	if ( ! $args['echo'] ) {
 		return $return;
 	}
-	echo $return;
+	echo wp_kses_post( $return );
```

Comment:
kses is probably overkill here since we're not filtering the HTML and generating it right above (and better to escape at that point).